outputs.tf

```diff
@@ -5,3 +5,7 @@ output "di_connectivity_tester_web_ip" {
 output "di_connectivity_tester_db_ip" {
   value = "${var.di_connectivity_tester_db_ip}"
 }
+
+output "data_ingest_iam_role" {
```

Comment:
can we pick a standard output that we can use everywhere
maybe `iam_roles` and always be a list (even if its just one, so that we have consistency)
means that the parent can then always use `module.module_name.iam_roles` without having to go see what it is, and whether its a list or string